modules/lb-vshn-hiera/git.tf

```diff
@@ -0,0 +1,9 @@
+module "git" {
```

Comment:
The indirection in this module seems overengineered, as far as I can see module "lb-vshn-hiera" doesn't do anything but configure it's "git" submodule. Why not just have those resources in this module instead?